libraries/backend/helpers.rb

```diff
@@ -92,6 +92,7 @@ class AzureEnvironments
       active_directory_endpoint_url: MicrosoftRestAzure::AzureEnvironments::AzureCloud.active_directory_endpoint_url,
       storage_endpoint_suffix: MicrosoftRestAzure::AzureEnvironments::AzureCloud.storage_endpoint_suffix,
       key_vault_dns_suffix: MicrosoftRestAzure::AzureEnvironments::AzureCloud.key_vault_dns_suffix,
+      datalake_store_filesystem_endpoint_suffix: MicrosoftRestAzure::AzureEnvironments::AzureCloud.datalake_store_filesystem_endpoint_suffix,
```

Comment:
these changes is not relevant to this PR

Reply:
Yes Sateesh.I think I add wrong file .That was related to data lake storage file system I was working on.Will remove this change.